Dependencies of v1-meta and v1-route-convention block upgrade to v2

### What version of Remix are you using?

2.0.0 (or trying to)

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

``` npx upgrade-remix 2.0.0```

On a 1.19.3 project with v1-meta and v1-route-convention both installed.


### Expected Behavior

Upgrade goes ahead.

### Actual Behavior

npm install fails:

```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: reset-apollo@undefined
npm WARN Found: @remix-run/dev@1.19.3
npm WARN node_modules/@remix-run/dev
npm WARN   peer @remix-run/dev@"^1.15.0" from @remix-run/v1-route-convention@0.1.2
npm WARN   node_modules/@remix-run/v1-route-convention
npm WARN     @remix-run/v1-route-convention@"^0.1.2" from the root project
npm WARN   1 more (the root project)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer @remix-run/dev@"^1.15.0" from @remix-run/v1-route-convention@0.1.2
npm WARN node_modules/@remix-run/v1-route-convention
npm WARN   @remix-run/v1-route-convention@"^0.1.2" from the root project
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @remix-run/v1-meta@0.1.1
npm ERR! Found: @remix-run/react@2.0.0
npm ERR! node_modules/@remix-run/react
npm ERR!   @remix-run/react@"2.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @remix-run/react@"^1.15.0" from @remix-run/v1-meta@0.1.1
npm ERR! node_modules/@remix-run/v1-meta
npm ERR!   @remix-run/v1-meta@"^0.1.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @remix-run/react@1.19.3
npm ERR! node_modules/@remix-run/react
npm ERR!   peer @remix-run/react@"^1.15.0" from @remix-run/v1-meta@0.1.1
npm ERR!   node_modules/@remix-run/v1-meta
npm ERR!     @remix-run/v1-meta@"^0.1.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

--- a/.changeset/eighty-llamas-bathe.md
+++ b/.changeset/eighty-llamas-bathe.md
@@ -1,0 +1,6 @@
+---
+"@remix-run/v1-route-convention": patch
+"@remix-run/v1-meta": patch
+---
+
+add remix v2 to peer dependencies

--- a/packages/v1-meta/package.json
+++ b/packages/v1-meta/package.json
@@ -31,8 +31,8 @@
     "react-dom": "^18.2.0"
   },
   "peerDependencies": {
-    "@remix-run/react": "^1.15.0",
-    "@remix-run/server-runtime": "^1.15.0"
+    "@remix-run/react": "^1.15.0 || ^2.0.0",
+    "@remix-run/server-runtime": "^1.15.0 || ^2.0.0"
   },
   "files": [
     "dist",

--- a/packages/v1-route-convention/package.json
+++ b/packages/v1-route-convention/package.json
@@ -30,7 +30,7 @@
     "@remix-run/dev": "^1.15.0"
   },
   "peerDependencies": {
-    "@remix-run/dev": "^1.15.0"
+    "@remix-run/dev": "^1.15.0 || ^2.0.0"
   },
   "files": [
     "dist",

